routes/web/project/postgres.rb

```diff
@@ -9,7 +9,8 @@ class CloverWeb
     end
 
     r.post true do
-      route_helper.post
+      route_helper.instance_variable_set(:@location, LocationNameConverter.to_internal_name(r.params["location"]))
+      route_helper.post(name: r.params["name"])
```

Comment:
If we pass `name: r.params.delete("name")` there, we don't need to add separate `required_parameters << "name" << "location" if @mode == AppMode::WEB` brach for web. We validate name and location separately anyway

Reply:
I think more about it, I believe we might not want to modify the original `r.params`. It's up to you

Reply:
Yep, initially I did it like that, then probably due to similar train of thought as yours, wanted to keep original params :smile: